dex/testing/dcrdex/harness.sh

```diff
@@ -20,8 +20,8 @@ cat > "${DCRDEX_DATA_DIR}/build" <<EOF
 #!/usr/bin/env bash
 cd ${HARNESS_DIR}/../../../server/cmd/dcrdex/
 go build -o ${DCRDEX_DATA_DIR}/dcrdex -ldflags \
-    "-X 'decred.org/dcrdex/dex.testLockTimeTaker=1m' \
-    -X 'decred.org/dcrdex/dex.testLockTimeMaker=2m'"
+    "-X 'decred.org/dcrdex/dex.testLockTimeTaker=3m' \
+    -X 'decred.org/dcrdex/dex.testLockTimeMaker=6m'"
 EOF
```

Comment:
There is a `build-lock` script that does this right below this. maybe that should do 1m-2m default if this is the new default. I think the shorter times are for some trade tests.